spec/lib/video_info/providers/youtube_spec.rb

```diff
@@ -487,4 +488,36 @@
       it { is_expected.to eq 'Deafheaven - Sunbather' }
     end
   end
+
+  context "with valid video and connection timeout" do
+    subject { VideoInfo.new('https://www.youtube.com/watch?v=lExm5LELpP4') }
+
+    describe '#title' do
+      before do
+        @stubbed = stub_request(:get, "https://www.youtube.com/watch?v=lExm5LELpP4").to_timeout
```

Comment:
Line is too long. [95/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.